week-1/Homework/mandatory/4-tax.js

```diff
@@ -2,40 +2,57 @@
   SALES TAX
   =========
   A business requires a program that calculates how much sales tax to charge
-  Sales tax is 20% of the price of the product
-*/
+  Sales tax is 20% of the price of the product 
 
-function calculateSalesTax() {}
+  Remember that the prices must include the sales tax (hint: you already wrote a function for this!)*/
+
+const TAX_RATE = 0.2;
+
+function calculateSalesTax(productPrice) {
+  productPrice = productPrice + productPrice * TAX_RATE;
+  return productPrice;
+}
+console.log(calculateSalesTax);
 
 /*
-  CURRENCY FORMATTING
+  CURRENCY FORMATTING 
   ===================
   The business has informed you that prices must have 2 decimal places
   They must also start with the currency symbol
-  Write a function that transforms numbers into the format £0.00
+  Write a function that transforms numbers into the format £0.00 */
 
-  Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
-*/
-
-function formatCurrency() {}
+function formatCurrency(productPrice) {
+  var productPriceTwo = productPrice + productPrice * 0.2;
+  return "£" + productPriceTwo.toFixed(2);
+}
```

Comment:
You did well at the 'toFixed' method, however you should read the instruction carefully. The function above does the calculation of the tax for you, you just need to put it inside for the formatCurrency function.